config/management/src/json_rpc.rs

```diff
@@ -48,6 +49,48 @@ impl JsonRpcClientWrapper {
         .ok_or_else(|| Error::JsonRpcReadError("validator-config", "not present".to_string()))
     }
 
+    /// This method returns all validator infos currently registered in the validator set of the
+    /// Libra blockchain. If account is specified, only a single validator info is returned: the
+    /// one that matches the given account.
+    pub fn validator_set(
```

Comment:
if we only use this logic once, maybe it is better to figure out how to get it into the actual place where it is being invoked instead of having these tiny command files above?

Reply:
Yeah, we should work on this in a cleanup PR.